### PR TITLE
devops: determine release update branch automatically

### PR DESCRIPTION
**Pull request by mxschmitt**
We are always interested in only the last release-* branch to update with our stable playwright.dev docs, hence we can automate the process of finding the correct version. This also allows us to run it scheduled, which makes it easier to keep cherry-picked docs changes in sync with the actual deployed version.

### CODE REVIEW

File: .github/workflows/roll.yml [hunk 1]
```diff
@@ -29,7 +30,7 @@ jobs:
       - name: Prepare branch
         id: prepare-branch
         run: |
-          BRANCH_NAME="roll/$(date +"%d-%m-%y")"
+          BRANCH_NAME="roll/next-$(date +"%d-%m-%y")"
```
Comment (mxschmitt) [inline .github/workflows/roll.yml:33]: to make sure we don't conflict with the other workflow.

File: .github/workflows/roll-next.yml [hunk 2]
```diff
@@ -29,7 +30,7 @@ jobs:
       - name: Prepare branch
         id: prepare-branch
         run: |
-          BRANCH_NAME="roll/$(date +"%d-%m-%y")"
```
Comment (yury-s) [inline .github/workflows/roll-next.yml:32]: Is this related?

**Review by yury-s — APPROVED** [review]

File: .github/workflows/roll-next.yml (hunk 2 above)
Comment (mxschmitt) [reply .github/workflows/roll-next.yml:32]: Yes, to make sure these two automated workflows don't clash with each-other. Since both use the same branch name format.
